src/com/google/javascript/jscomp/ProcessCommonJSModules.java

```diff
@@ -666,7 +666,7 @@ private void visitExport(NodeTraversal t, Node export) {
       }
 
       if (root.getParent().isAssign()
-          && (root.getNext().isName() || root.getNext().isGetProp())
+          && (root.getNext() != null && (root.getNext().isName() || root.getNext().isGetProp()))
```

Comment:
> I implemented a subset of the browser key in package.json as an overlay. Is this something you had on your radar?

No it wasn't. You referring to this "spec"? https://github.com/defunctzombie/package-browser-field-spec